staging/src/k8s.io/api/go.mod

```diff
@@ -38,16 +37,7 @@ require (
 )
 
 replace (
-	github.com/google/cadvisor => github.com/openshift/google-cadvisor v0.47.3-openshift-4.15-1
```

Comment:
This shouldn't have happened.  I'm going to try to pull locally and see if I get a different result.